packages/account-v2/src/containers/DocumentSelection/DocumentSelection.tsx

```diff
@@ -1,44 +1,60 @@
 import React from 'react';
 import { StandaloneChevronRightRegularIcon } from '@deriv/quill-icons';
-import { Text, useDevice } from '@deriv-com/ui';
+import { Button, Divider, Text, useDevice } from '@deriv-com/ui';
 import { Card } from '../../components/Card';
 import { getManualUploadDocumentList } from '../../constants/manualDocument';
 
 type TDocumentSelection = {
     countryCode: string;
     handleOnClick: (value: string) => void;
+    onCancel: () => void;
 };
 
-export const DocumentSelection = ({ countryCode, handleOnClick }: TDocumentSelection) => {
+export const DocumentSelection = ({ countryCode, handleOnClick, onCancel }: TDocumentSelection) => {
     const { isMobile } = useDevice();
 
     const documentList = getManualUploadDocumentList(countryCode === 'ng');
 
     return (
-        <div className='flex flex-col'>
-            <Text as='h2' color='prominent' size='xs'>
-                Please upload one of the following documents:
-            </Text>
-            {documentList.map(({ description, icon: Icon, title, value }) => (
-                <Card
-                    className='flex flex-row items-center cursor-pointer'
-                    data-testid='dt_manual_document_types'
-                    key={value}
-                    onClick={() => handleOnClick(value)}
+        <div className='grid h-full'>
+            <section className='flex flex-col gap-18'>
+                <Text as='h2' className='mt-32' color='prominent' size='sm'>
+                    Please upload one of the following documents:
+                </Text>
+                {documentList.map(({ description, icon: Icon, title, value }) => (
```

Comment:
Hi @likhith-deriv 👋 
By any chance can we break this in multipule component ? 👀 

Thank you 🙏

Reply:
How do u suggest we do that? Its basically Header, Document list and buttons